client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -2,9 +2,9 @@
  * External dependencies
  */
 import { __ as NO__ } from '@wordpress/i18n';
-import { SelectControl } from '@wordpress/components';
+import { FormTokenField, SelectControl } from '@wordpress/components';
```

Comment:
`FormTokenField` does a pretty good job of offering suggestions based on typing and our verticals. But it's really designed to add multiple discrete inputs, like tags for a post.

It would be nice to have the suggetion + autocomplete dropdown functionality. That functionality seems to exist in the `FormTokenField`, but not on its own.